mmpose/datasets/datasets/body3d/body3d_h36m_dataset.py

```diff
@@ -80,6 +80,8 @@ def load_config(self, data_cfg):
             self.camera_param = self._load_camera_param(
                 data_cfg['camera_param_file'])
 
+        self.subjects = data_cfg.get('subjects', None)
```

Comment:
The subject/action filter was also added in another PR (https://github.com/open-mmlab/mmpose/blob/feb6d668f351016f6f6a48ce34368fac29e110ce/mmpose/datasets/datasets/body3d/body3d_h36m_dataset.py#L97).

It's okay to leave it and we will solve the conflict later.